src/arch/x86/32/machine/capdl.c

```diff
@@ -12,6 +12,46 @@
 #include <machine/capdl.h>
 #include <arch/machine/capdl.h>
 
+word_t get_tcb_sp(tcb_t *tcb)
+{
+    return tcb->tcbArch.tcbContext.registers[ESP];
+}
+
+void print_ipc_buffer_slot(tcb_t *tcb)
+{
+#ifdef CONFIG_PRINTING
```

Comment:
is this define really necessary? `printf()` should be defined to nothing if `CONFIG_PRINTING this is not set. And in one of my other PR the feedback was that in this cases there should be no define guard around this then.

Reply:
I think Axel is right, this is already outside the verification build so can't disturb anything there, and `printf` itself is `#ifdef`'d, so we should drop all the `#ifdef CONFIG_PRINTING` in this patch.

Reply:
Sounds good.